x-pack/test/functional/apps/uptime/monitor.ts

```diff
@@ -12,13 +12,11 @@ export default ({ getPageObjects, getService }: FtrProviderContext) => {
   const archive = 'uptime/full_heartbeat';
 
   describe('monitor page', function() {
-    this.tags(['skipFirefox']);
```

Comment:
I think we should leave these as skipped in firefox, no? If we don't need them then we can keep the firefox tests slim without them.

Reply:
Ah ok, this was a misunderstanding on my part.

Reply:
I added https://github.com/elastic/kibana/pull/46269/commits/b91fd55154eb1a0208270711dcb76aa63f7f810b to reinsert that.